core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -635,29 +649,28 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         }
     }
 
-    function _safeUintCast(int256 value) private pure returns (uint256 result) {
```

Comment:
I removed this function because it is redundant; the if statement in the function that calls this means that it will never get used. This saves us ~100 bytes